src/display/editor/draw.js

```diff
@@ -71,10 +71,20 @@ class DrawingEditor extends AnnotationEditor {
 
   static _currentDraw = null;
 
+  static _currentDrawingAC = null;
```

Comment:
For all the new fields added in the patch, here and below, could we make them actually private rather than just semi-private?
```suggestion
  static #currentDrawingAC = null;
```